app/src/main/java/org/p2p/wallet/auth/model/PhoneNumber.kt

```diff
@@ -0,0 +1,20 @@
+package org.p2p.wallet.auth.model
+
+data class PhoneNumber(val value: String) {
```

Comment:
from the name of the class i can't understand how the value looks inside
is it +7 123 456 78 90 or +71234567890 or 71234567890